src/Kovan-DssSpell.t.sol

```diff
@@ -12,13 +12,18 @@ interface Hevm {
     function store(address,bytes32,bytes32) external;
 }
 
+interface MedianizerV1Abstract {
+    function authority() external view returns (address);
+    function owner() external view returns (address);
+    function peek() external view returns (uint256, bool);
+    function poke() external;
+}
+
```

Comment:
Does this really need to be added?

Reply:
We just needed `authority()`, removed others.